docs-next/cypress/integration/general_spec.js

```diff
@@ -0,0 +1,27 @@
+describe('all pages', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:8000');
+  });
+  it.skip('should have basic markup', () => {
+    return cy.get('nav a').each(navItem => {
+      cy.visit(navItem[0].href).get('nav');
+      cy.get('header');
+      cy.get('main');
+    });
+  });
+  it('shouldn not have any empty links', () => {
+    return cy.get('nav a').each(navItem => {
+      cy.visit(navItem[0].href);
+      cy.document().then(document => {
```

Comment:
cy.get() will throw if it times out on a query. 
In this case we want to run an assertion if anchors DO exist, and not fail if they don't. 

I hate this because there's a LOT of ways this can go wrong, but according to cypress docs this is probably the most reliable way to make this kind of conditional assertion.